<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string> Test Times:    </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string> Test Gap(ms):</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edit_div">
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="edit_times">
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="text">
     <property name="text">
      <string> Package Name:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="btn_app">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Choose App:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QTextEdit" name="textEdit_apppath">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QTextEdit" name="textEdit_packagename">
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
